release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1278,6 +1278,26 @@ sourceStrategy:
 [id="ocp-4-11-installer-bug-fixes"]
 ==== Installer
 
+* Previously, if users specified an {product-title} cluster name with a dot (.), the installation failed. This update adds a validation check to the installation program, which returns an error if a dot is present in the cluster name. link:https://bugzilla.redhat.com/show_bug.cgi?id=2084580[*BZ#2084580*]
+
+* Previously, users could select the AWS `us-gov-east-1` region when using the installation program to create the `install-config.yaml` file. This caused the deployment to fail because the installation program could only be used to create the `install-config.yaml` file for a public AWS region. This update removes all of the AWS regions from the installation program that are not supported by the public AWS cloud. link:https://bugzilla.redhat.com/show_bug.cgi?id=2048222[*BZ#2048222*]
+
+* Previously, users could not select the `ap-north-east-3` region when using the installation program to create the `install-config.yaml` file. The AWS SDK that caused this problem has been updated, and users can now select the `ap-north-east-3` region. link:https://bugzilla.redhat.com/show_bug.cgi?id=1996544[*BZ#1996544*]
+
+* Previously, installing a private (internal) {product-title} cluster on Azure Stack Hub failed because the installation program did not create the DNS record for the API VIP. This update removes the invalid check that caused this problem. The installation program now correctly creates DNS records for private cluster. link:https://bugzilla.redhat.com/show_bug.cgi?id=2061549[*BZ#2061549*]
```

Comment:
```suggestion
* Previously, installing a private (internal) {product-title} cluster on Azure Stack Hub failed because the installation program did not create the DNS record for the API virtual IP address. This update removes the invalid check that caused this problem. The installation program now correctly creates DNS records for private cluster. link:https://bugzilla.redhat.com/show_bug.cgi?id=2061549[*BZ#2061549*]
```